boards/arm/nucleo_f207zg/nucleo_f207zg_defconfig

```diff
@@ -0,0 +1,38 @@
+CONFIG_ARM=y
+CONFIG_BOARD_NUCLEO_F207ZG=y
+CONFIG_SOC_SERIES_STM32F2X=y
+CONFIG_SOC_STM32F207XG=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=120000000
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
+
+CONFIG_SERIAL=y
+
+# console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# enable pinmux
+CONFIG_PINMUX=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# clock configuration
+CONFIG_CLOCK_CONTROL=y
+
+# Clock configuration for Cube Clock control driver
+CONFIG_CLOCK_STM32_HSE_CLOCK=8000000
+CONFIG_CLOCK_STM32_SYSCLK_SRC_PLL=y
+# use HSE as PLL input
+CONFIG_CLOCK_STM32_PLL_SRC_HSE=y
+# however, the board does not have an external oscillator, so just use
+# the 8MHz clock signal coming from integrated STLink
+CONFIG_CLOCK_STM32_HSE_BYPASS=y
+# produce 96MHz clock at PLL output
```

Comment:
Comment is wrong. I did the math and indeed this will result in the 120MHz mentioned further above.

Reply:
Yes it's wrong.  @erwango had report this issue seems I had forget change it.